osbuild/loop.py

```diff
@@ -333,6 +333,11 @@ def _config_info(self, info, offset, sizelimit, autoclear, partscan):
                 info.lo_flags |= self.LO_FLAGS_PARTSCAN
             else:
                 info.lo_flags &= ~self.LO_FLAGS_PARTSCAN
+        if read_only is not None:
+            if read_only:
```

Comment:
I think it's a bit more idiomatic in python to use keyword args for this? i.e. `read_only=False` above?

Oh but I see you're just following an existing pattern.